### ISSUE TITLE
List of repos affected by template repos

### ISSUE THREAD

**Issue by BenGitsCode** [ga-wdi-boston/rails-api-template#30]
per Chris "should we have a list of other repos that are affected by the other repos that will be affected by this one?"

This can/should be extended to all template-repos as a process improvement.

@ga-wdi-boston/core

**Comment by raq929** [comment]
Please edit this comment with repos affected by this one.
ga-wdi-boston/rails-api
ga-wdi-boston/rails-api-single

**Comment by gaand** [comment]
I think I disagree, quite strongly, with keeping this list.

Repositories that depend on this one may come and go.  Maintaining this list seems like unnecessary make work.

Updating from template is process.  The only thing that I think needs to change is ensuring we update the template repositories before we start updating the dependent repositories.

**Comment by gaand** [comment]
My answer to the revised title is "no".

**Comment by jrhorn424** [comment]
> My answer to the revised title is "no".

Agreed. Use the git history. If it isn't there, ask someone else (can't imagine why this would be the case).

I don't mind keeping a document that explains how templates are connected to concrete repos through our **naming strategies**, but an exhaustive list is a lot of work.

**Comment by jrhorn424** [comment]
That list may one day exist once we automate a curriculum manifest (I have a lot of reasons for doing that). In which case, it may serve us some utility as we prep, but until then I think our naming heuristics are enough.
